src/App.js

```diff
@@ -14,11 +15,108 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    baseOrder: [...goodsFromServer],
```

Comment:
```suggestion
```
no need for this. You can reset your order of goods by assigning [...goodsFromServer] to your 'products' field